conf/accumulo/common/accumulo-it.properties

```diff
@@ -13,3 +13,5 @@ accumulo.it.cluster.standalone.client.conf=ACCUMULO_HOME/conf
 accumulo.it.cluster.standalone.server.conf=ACCUMULO_HOME/conf
 accumulo.it.cluster.standalone.client.cmd.prefix=
 accumulo.it.cluster.standalone.server.cmd.prefix=
+accumulo.it.cluster.standalone.users.1=user1
```

Comment:
Why add these props?

Reply:
The tests will error without users defined

Reply:
So some Accumulo ITs expect the cluster to already have these user defined?  The ITs can not create the users?

Reply:
2 users are needed for the AccumuloClientIT